package.json

```diff
@@ -80,6 +80,7 @@
     "babel-traverse": "^6.26.0",
     "babel-types": "^6.26.0",
     "babylon": "^6.18.0",
+    "cancel-token": "^0.1.1",
```

Comment:
I'd like to see this library moved to PolymerLabs, or maybe even Polymer, since we're depending on it from non-labs code.

Reply:
Yeah, seems like things have cooled down a bit. Looks like the fetch cancellation API is basically this one, though with fewer methods on the token: https://developer.mozilla.org/en-US/docs/Web/API/AbortController

Reply:
FYI, I think it's fine to not exactly emulate any other API for the time being. Maybe just don't call it a polyfill.

Reply:
Moved to PolymerLabs, happy to move it from there to Polymer, maybe after a 1.0?

Updated a few links, README, and added a LICENSE file here: https://github.com/PolymerLabs/cancel-token/pull/2